desc/io/efit_io.py

```diff
@@ -0,0 +1,234 @@
+"""Utilities for reading EFIT geqdsk files."""
+
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.interpolate import CubicSpline
+
+
+def efit_to_desc(g, M, N=0, L=None, sep_dev=0):
```

Comment:
I would prefer if there were an option to specify current or iota profile, if not then specify in docstring it willl return a Fixed-iota equiibrium

Reply:
Right now it's just fixed iota because the eqdsk doesn't explicitly contain the current values. What we can do in the long run is to calculate the current numerically by using the p' and F' values.